stubs/pywin32/_win32typing.pyi

```diff
@@ -4801,7 +4801,10 @@ class PyCBitmap:
         arg1: tuple[Incomplete, Incomplete, Incomplete, Incomplete],
     ) -> None: ...
     def GetInfo(self): ...
-    def GetBitmapBits(self, asString: int = ...) -> str: ...
+    @overload
+    def GetBitmapBits(self, __asString: Literal[False] = False) -> tuple[int, ...]: ...
+    @overload
+    def GetBitmapBits(self, __asString: Literal[True]) -> bytes: ...
```

Comment:
Worth having a fallback overload, to avoid false positives if the type checker is unable to infer a `Literal`?

```suggestion
    def GetBitmapBits(self, __asString: Literal[True]) -> bytes: ...
    @overload
    def GetBitmapBits(self, __asString: bool) -> Any: ...
```

Reply:
Ah, thought it was managed by checkers for some reason.

Although I'm tempted to prefer the real union here: a typical use-case would use a literal. And if you take an unknown boolean parameter, you should really support both return types. There's only two possibilities here, so either you use a literal for the 1 suported case, or you support both.

```py
import win32ui
bitmap = win32ui.CreateBitmap()
# ...
def foo(as_bytes) -> bytes:
  return bitmap.GetBitmapBits(as_bytes) # This doesn't make sense
  
def foo(as_bytes) -> bytes: # Unused param
  return bitmap.GetBitmapBits(True) # This is more sensible
```

Reply:
I'm okay with going with a union return type for the fallback overload here. After all, it's the fallback, we hope the type checker generally doesn't hit it :)

> thought it was managed by checkers for some reason.

Type checkers do "`Union` math" for overloads in some situations, but generally don't when it comes to overloads that use `Literal` types. I think there's a good reason for that, but I can't give it to you off the top of my head I'm afraid :)